### PR TITLE
[PR] Add initial shortcode for embedding HTML components

### PR DESCRIPTION
**Pull request by jeremyfelt**
Note that if testing locally, you'll need to filter `allowed_html_component_url`. This prevents arbitrary HTML from being embedded.

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: plugin.php [hunk 1]
```diff
@@ -15,15 +15,11 @@
...
- * Start things up.
- *
- * @return \HTML_Component_Embed
- */
-function HTML_Component_Embed() {
-	return HTML_Component_Embed::get_instance();
+// This plugin uses namespaces and requires PHP 5.3 or greater.
+if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
+	add_action( 'admin_notices', create_function( '',
+	"echo '<div class=\"error\"><p>" . __( 'WSUWP Plugin Skeleton requires PHP 5.3 to function properly. Please upgrade PHP or deactivate the plugin.', 'wsuwp-plugin-skeleton' ) . "</p></div>';" ) );
```
Comment (philcable) [inline plugin.php]: s/WSUWP Plugin Skeleton/HTML Component Embed :)

**Review by philcable — APPROVED** [review]
👍  Looks great! I left one inline comment, just about a reference to Plugin Skeleton.

File: plugin.php (hunk 1 above)
Comment (jeremyfelt) [reply plugin.php]: hahaha, I need to write a script to replace all of those :)

**Pull request merged**